src/vs/platform/telemetry/common/telemetryService.ts

```diff
@@ -110,15 +110,15 @@ export class TelemetryService implements ITelemetryService {
 
 		return this._commonProperties.then(values => {
 
-			// (first) add common properties
-			data = mixin(data, values);
-
 			// (next) add experiment properties
 			data = mixin(data, this._experimentProperties);
 
 			// (last) remove all PII from data
 			data = cleanData(data as Record<string, any>, this._cleanupPatterns);
 
+			// (first) add common properties
```

Comment:
comments are out of date

Reply:
I'm not sure why we decided to comment the order, the order is clear in the source code.